templates/snyk-cloud.template.yaml

```diff
@@ -0,0 +1,139 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Snyk Cloud IAM Role #TODO add tracking ID
```

Comment:
Noting a TODO for the AWS team.

Reply:
Resolved in [382481a](https://github.com/aws-quickstart/quickstart-snyk-security/pull/47/commits/382481a7bae7a8b29fd8938f3c3c0bf7465fa3c8)